src/Cake.Common/Tools/NuGet/List/NuGetListSettings.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using Cake.Core.IO;
+using Cake.Core.Tooling;
+
+namespace Cake.Common.Tools.NuGet.List
+{
+    /// <summary>
+    /// Contains settings used by <see cref="NuGetList"/>.
+    /// </summary>
+    public sealed class NuGetListSettings : ToolSettings
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether list all versions of a package.By default, only the latest package version is displayed
+        /// </summary>
+        public bool AllVersions { get; set; }
+
+        /// <summary>
+        /// Gets or sets the amount of details in the output: normal, quiet, detailed.
+        /// </summary>
+        public NuGetVerbosity? Verbosity { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to allow prerelease packages to be shown.
+        /// </summary>
+        public bool Prerelease { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to allow unlisted packages to be shown.
+        /// </summary>
+        public bool IncludeDelisted { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to force the application to run using an invariant, English-based culture.
+        /// </summary>
+        public bool ForceEnglishOuput { get; set; }
```

Comment:
Is this needed?

Reply:
No and if it's needed for parsing to be consistent across cultures then it should always be explicitly set.

Reply:
ForceEnglishOuput has been removed